cmiles/utils.py

```diff
@@ -77,44 +89,73 @@ def load_molecule(inp_molecule, backend='openeye'):
         If has license to OpenEye, will return an OpenEye molecule. Otherwise will return a RDKit molecule if input can
         be parsed with RDKit.
     """
-    from rdkit import Chem
-    try:
-        from openeye import oechem
-        has_openeye = True
-    except ImportError:
-        has_openeye = False
 
+    if backend == 'rdkit':
+        if not has_rdkit:
+            raise RuntimeError("You need to have RDKit installed to use the RDKit backend")
+        molecule = _load_mol_rd(inp_molecule)
+
+    if backend == 'openeye':
```

Comment:
Probably best handled with a `if/elif/else` here to catch the case if `back = "garbage"`.